### ISSUE TITLE
Support variable stack names

### ISSUE THREAD

**Issue by tgass** [zalando/lizzy-client#103]
*Given:*
In my senza definition file, the StackName is defined using an Arguments-variable:
```
StackName: "wps{{Arguments.StackNamePostfix}}"
```

Unfortunately, lizzy doesn't resolve the argument. I get the following error:

```
Requesting new stack.. EXCEPTION OCCURRED: 404 Client Error: NOT FOUND for url: https://lizzy-zprice-test.zprice-test.zalan.do/api/stacks

[AGENT] Stack not found: wps{{Arguments.StackNamePostfix}}-74ee2ddxb35

FAILURE: Build failed with an exception.
```

I'd like to see that lizzy can handle variables in stack names.

PS
I suppose this can be handle in the client.

**Comment by dcherepovsky** [comment]
@tgass Of course, it can be implemented on either side of lizzy. But for me it looks, that this is a feature of [lizzy-agent](https://github.com/zalando/lizzy), not client.

**Comment by adyach** [comment]
the same issue here. upvote.

**Comment by bugzhunter** [comment]
we had the same issue, upvoted

**Comment by SergKam** [comment]
should be fixed here https://github.com/zalando-stups/lizzy/issues/216
